### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by vuqtran88**
update change log for expected error.

### CODE REVIEW

File: src/Agent/CHANGELOG.md [hunk 1]
```diff
@@ -17,7 +23,9 @@ Fixes an issue where Garbage Collection Performance Metrics may not be reported
 Fixes an issue where `newrelic.config` was being overwritten when upgrading the agent via either `rpm`/`yum` (RedHat/Centos) or `dpkg`/`apt` (Debian/Ubuntu).
  
 ### Docs Changes
-* [Doc Title](https://urlToDraft)
+* https://docs.newrelic.com/node/13816/draft
```
Comment (alanwest) [inline src/Agent/CHANGELOG.md:26]: Need to remove these draft links prior to release.

File: src/Agent/CHANGELOG.md [hunk 2]
```diff
@@ -9,16 +9,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * **The .NET Agent is now open source!** <br/>
 The New Relic .NET agent is now open source! Now you can view the source code to help with troubleshooting, observe the project roadmap, and file issues directly in this repository.  We are now using the [Apache 2 license](/LICENSE). See our [Contributing guide](/CONTRIBUTING.md) and [Code of Conduct](/CODE_OF_CONDUCT.md) for details on contributing!
 
+* **Expected Errors Support** <br/>
+Certain errors that are expected within the application may be identified so that they will not be counted towards the application's error rate and Apdex Score.  Only errors that truly affect the health of the application will be alerted on.  Please review the following [documentation](https://docs.newrelic.com/docs/agents/net-agent/configuration/net-agent-configuration#error_collector) for details on how to configure Expected Errors.
+
+* **Ignored Errors Enhancements** <br/>
```
Comment (alanwest) [inline src/Agent/CHANGELOG.md:15]: What is the enhancement we're highlighting? Ignored errors have been around for along time. I know there were changes, but they are not described here.

File: src/Agent/CHANGELOG.md [hunk 3]
```diff
@@ -9,16 +9,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+Certain errors may be identified in configuration so that they will be ignored.  These errors will not be counted towards the application's error rate, Apdex score, and will not be reported by the agent. Please review the following [documentation](https://docs.newrelic.com/docs/agents/net-agent/configuration/net-agent-configuration#error_collector) for details on how to configure Ignored Errors.
+    * New configuration element [`<ignoreMessages>`](https://docs.newrelic.com/docs/agents/net-agent/configuration/net-agent-configuration#error-ignoreErrors)supports filtering based on the error message. 
+    * Please note that the [`<ignoreErrors>`](https://docs.newrelic.com/docs/agents/net-agent/configuration/net-agent-configuration#error-ignoreErrors) configuration element has been deprecated and replaced by [`<ignoreClasses>`](https://docs.newrelic.com/docs/agents/net-agent/configuration/net-agent-configuration#error-ignoreClasses).  The .NET Agent continures to support this configuration element, but its support may be removed in the future.
```
Comment (alanwest) [inline src/Agent/CHANGELOG.md:18]: continures -> continues

The .NET Agent still supports `<ignoreErrors>`, but it may be removed in the future.

**Comment by jifeingo** [comment]
CI retest this

**Comment by jifeingo** [comment]
CI retest this

**Comment by jifeingo** [comment]
jenkins retest this

**Pull request merged**